src/plugins/httpUpload/HttpUploadPlugin.ts

```diff
@@ -254,6 +255,32 @@ export default class HttpUploadPlugin extends AbstractPlugin {
             }
          }
       }
+      else
+          if (body.length===1 && !message.isEncrypted())
+          {
+              var link = body.text();
```

Comment:
Url with text will not be detected.

Reply:
is see jsxc will send urls with text in the following format:
url\ntext...
Now this will be parsed too.
Furthermore urls with example: http://domain:port/image.jpg?parameter... parameters will be parsed correctly too